packages/reports/addon/routes/reports/report.js

```diff
@@ -208,6 +209,24 @@ export default Route.extend({
       controller.set('reportState', state);
     },
 
+    /*
+     * @action setHasRequestRun
+     * @param {Boolean} value
+     */
+    setHasRequestRun(value) {
+      const controller = this.controllerFor(this.routeName);
+      controller.set('hasRequestRun', value);
```

Comment:
Is possible to compare the model's request to the current request to see if the current request has run?

Reply:
@jkusa so I've tried a few things
1. a property in the `view` controller, but can't think of a way to access the request of the parent model without setting it explicitly.
something like `computed('reportController.model.request' ...` or a cp `reportController.request` doesn’t seem to trigger any change
2. pass down `request` along with `response` to `view.hbs` and have a cp there that compares it with `report.request` - also doesn't work

either I'm missing something or the explicit `set` of the request was there for a reason, will try to think of another way to approach this.

In any case we _can_ get rid of `hasRequestRun` while still setting the request explicitly from `onUpdateReport`, I just think `hasRequestRun` might be useful to other child routes in the future